README.md

```diff
@@ -55,6 +56,11 @@ Before you get started, the following needs to be installed:
   bundle install
   ```
 
+1. Install node modules
+  ```bash
+  npm install
```

Comment:
This seems to take a lifetime and quickly looking at the list includes modules that are not in active use. I would suggest pruning this to minimum now and adding later more modules if needed (things like react router).